### PR TITLE
docs: create how-runme-works subsection

### PR DESCRIPTION
**Pull request by edeediong**

### CODE REVIEW

File: docs/getting-started/faq.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## **How do you plan to make money with Runme? Will you introduce paid enterprise or support plans, or consider monetizing user data?**
+
+Currently, we don't have specific plans to charge for Runme as it's available under the open-source APL-2.0 license. Stateful, Inc., the company behind Runme, is initially focusing on building open technology to tackle issues with outdated developer documentation. We're committed to keeping Runme as an open-source solution, delivering powerful features to the community.
+
+Our goal is to create a community around addressing documentation challenges and maintaining a trusted relationship with our users. We want to assure you that we won't monetize or sell users' data under any circumstances. Potential monetization opportunities may arise from providing additional value in terms of team collaboration, discoverability, or auditing features for companies using Runme with internal repositories. The core functionality of Runme, such as running runbooks, will remain available without the need for proprietary products from Runme, Stateful, or third parties.
+
+## **What is Runme Default editor in Vscode?**
```
Comment (sourishkrout) [inline docs/getting-started/faq.md:22]: `VS Code` brand spelling

File: docs/how-runme-works/VSCode.md [hunk 2]
```diff
@@ -8,29 +8,27 @@ title: Runme for VS Code
 
 import Infobox from "../../src/components/Infobox.tsx"
 
-# Getting Started
+# Runme on Vscode
```
Comment (sourishkrout) [inline docs/how-runme-works/VSCode.md:11]: `VS Code` brand spelling

File: docs/how-runme-works/cli.md [hunk 3]
```diff
@@ -22,13 +21,13 @@ runme
...
+his allows you to quickly execute any script present in the README.
 
 <Infobox type="warning" title="Warning!">
 
 Runme won't work if your current working directory doesn't contain a README file at the top level!
 
-You can use the `--chdir` flag to alter the working directory or `--filename` to specify any other markdown file, without changing the environment's working directory.
+You can use the `--chdir` flag to alter the working directory or `--filename` to specify any other markdown file, without changing the environment's working directory.
 
 </Infobox>
```
Comment (sourishkrout) [inline docs/how-runme-works/cli.md:33]: The help text below is old

**Review by sourishkrout — CHANGES_REQUESTED** [review]
See comments please. Also, "How Runme Works" needs below "Getting Started" in the side navbar. It's out of place in 3rd place.

File: docs/how-runme-works/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+sidebar_position: 3
+title: How Runme Works
+---
+
+import Infobox from "../../src/components/Infobox.tsx"
+import EnvironmentOptions from "../../src/components/EnvironmentOptions.tsx"
+
+# How Runme Works
+
+Runme aims to deliver an excellent experience from the very beginning. To learn how to make the most out of our markdown docs. If you haven't already installed Runme, please complete the steps in [installation](https://docs.runme.dev/install#runme-cli) before proceeding.
```
Comment (joseeantonior) [inline docs/how-runme-works/index.md:11]: This line doesn't quite make sense to me. Please review and reword.

**Review by joseeantonior — CHANGES_REQUESTED** [review]
- "Using runme on a remote server with VSCode" let's capitalize Runme and add a space in VS Code. // Runme via SSH

**Comment by joseeantonior** [comment]
As a side-note, we should grep all of the appearances of `vscode` and `markdown` and make sure they are properly spelled.

**Review by sourishkrout — APPROVED** [review]
👍  LGTM

**Pull request merged**